libvast/src/detail/process.cpp

```diff
@@ -53,6 +56,8 @@ caf::expected<path> objectpath_static() {
 } // namespace
 
 caf::expected<path> objectpath(const void* addr) {
+  if (addr == nullptr)
+    addr = (const void*) objectpath_dynamic;
```

Comment:
Is there a specific reason for the C-style cast?

Reply:
I changed it to `reinterpret_cast`.